app/src/main/res/values/styles.xml

```diff
@@ -22,7 +22,7 @@
         <item name="color">@android:color/white</item>
     </style>
 
-    <style name="AppTheme.Button" parent="Widget.AppCompat.Button">
+    <style name="AppTheme.Button" parent="Widget.MaterialComponents.Button">
```

Comment:
can we rename it to something better, something prefixed with `Material`, should be suffice

Reply:
okay, I'll change it to `Material.Button`